lib/queue.js

```diff
@@ -26,16 +28,9 @@ module.exports = FetchQueue;
 
 FetchQueue.prototype = [];
 FetchQueue.prototype.add = function(protocol, domain, port, path, depth, callback) {
```

Comment:
I wonder if we need a handy public method for constructing a queueItem from a URL, in that case? So you'd do something like:

```js
crawler.queue.add(crawler.queue.createQueueItem("http://foo.com:8080/bar?baz"), function() {
    // stuff here
});
```